src/App.js

```diff
@@ -2,12 +2,60 @@ import React from "react";
 import "./App.css";
 
 class App extends React.Component {
+  constructor () {
+    super ()
+    this.state = {
+      score: 0,
+      currentValue: 1,
+      nextValue: 2,
+    }
+  }
+
+  increaseScore = () => {
+    this.setState({ score: this.state.score + this.state.currentValue });
+  }
+
+  message = () => {
+    if (this.state.score < 10){
+        alert("You can't afford that!");
+    } else {
+    this.setState({
+      score: this.state.score - 10,
+      currentValue: this.state.currentValue + 1,
+      nextValue: this.state.nextValue + 1,
+      });
+    }
+  }
+
+reset = () => {
```

Comment:
Remember to keep things simple. you could reset the state as seen below. 

`this.setState({
      score: 0,
      currentValue: 1,
      nextValue: 2,
    })`